CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+### v2.0.0-beta3 -> v2.0.0-beta4
+
+#### Features
```

Comment:
> should we even be PRing this change?

It's a good point--this CHANGELOG update is just a single housekeeping PR, which was relevant for the previous beta. I would argue it's still informative to do CHANGELOG updates everywhere, even if they literally contained nothing but the header, so readers know there were no changes rather than wondering if we forgot to update the CHANGELOG.